sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2148,6 +2148,16 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val ADD_PARTITION_BATCH_SIZE =
+    buildConf("spark.sql.addPartitionInBatch.size")
+      .internal()
+      .doc("The number of partitions to be handled in one turn when use " +
+        "`AlterTableAddPartitionCommand` to add partitions into table. The smaller " +
```

Comment:
We should avoid using the internal class name in the conf description and log/exception messages, since most readers are external end users who have no idea about the internal code.

Reply:
This is an internal config, only Spark developers would read it.

Reply:
Most Spark developers do not know INSERT also can add partitions if the partitions do not exist. The current description is not clear.